.github/workflows/gt4py-sphinx.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       with:
         python-version: 3.8
     - name: Install dependencies
+      env:
+      PIP_EXTRA_INDEX_URL: https://test.pypi.org/simple
```

Comment:
We should remove PIP_EXTRA_INDEX_URL before merging to avoid security issues and using unstable versions by mistake (gridtools_cpp should have been already released in the official PyPI at that point).

Reply:
For reference: the same env variable should be removed from `tox.ini` as soon as the official release is made.